internal/pkg/mapper/mapper_test.go

```diff
@@ -0,0 +1,88 @@
+package mapper
+
+import (
+	"fmt"
+
+	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	aggregationv1 "github.com/envoyproxy/xds-relay/pkg/api/aggregation/v1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+type KeyerConfiguration = aggregationv1.KeyerConfiguration
+type Fragment = aggregationv1.KeyerConfiguration_Fragment
+type FragmentRule = aggregationv1.KeyerConfiguration_Fragment_Rule
+type MatchPredicate = aggregationv1.MatchPredicate
+type ResultPredicate = aggregationv1.ResultPredicate
+
+const (
+	clusterTypeURL = "type.googleapis.com/envoy.api.v2.Cluster"
+	nodeid         = "nodeid"
+	nodecluster    = "cluster"
+	noderegion     = "region"
+	nodezone       = "zone"
+	nodesubzone    = "subzone"
+	stringfragment = "stringfragment"
+)
+
+var _ = Describe("GetKey", func() {
+	DescribeTable("should be able to return error",
+		func(match *MatchPredicate, result *ResultPredicate, typeurl string, assert string) {
```

Comment:
nit: camelcase typeurl

Reply:
yeah...the go linter likes it this way.

Reply:
![image](https://user-images.githubusercontent.com/49211422/76643838-1471ae80-6513-11ea-867c-e71f613fef27.png)

Reply:
typeURL is fine too, but does it enforce all lowercase in that test function?

Reply:
which function are you referring to?

Reply:
The one for this GitHub comment (L32)

Reply:
it's not a big deal

Reply:
![image](https://user-images.githubusercontent.com/49211422/76645516-610ab900-6516-11ea-8002-7596a58c2f8b.png)

Reply:
may be specific to my vscode only?

Reply:
yeah `typeURL` would be the go preferred option

Reply:
i think as long as the `make lint` goes through, we should be ok

Reply:
i understand its hard to catch such typos, so lets lean in to the linter and do what it tells